lib/colors.js

```diff
@@ -115,6 +115,13 @@ function applyTheme (theme) {
   for (var style in theme) {
     (function(style){
       colors[style] = function(str){
+        if (typeof theme[style] === 'object'){
```

Comment:
Both work, in the demo, but this allows Objects as associative arrays: `{cool: 'underline', funky: 'blue'}`. I don't know why this would ever be needed, maybe `Array.isArray` is better.